src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+    text-decoration: none;
+    min-height: 200px;
+    overflow: auto;
+    display: grid;
+    margin-top: 66px;
+    opacity: 1;
+}
+
+.rectangle {
+    grid-row: 1;
+    grid-column: 1;
+    opacity: 0.3;
+    background-color: #2A2C2F;
+    z-index: 1;
+}
+
+.wrapper:hover .rectangle {
+    transition: opacity .4s;
+    opacity: 0.8;
+}
+
+.coverPhoto {
+    grid-row: 1;
+    grid-column: 1;
+width: 100%;
+height: 100%;
+z-index:0;
```

Comment:
Usually you do not need `z-index` without `position: absolute`

Reply:
same as above